eras/babbage/impl/src/Cardano/Ledger/Babbage/Rules/Utxo.hs

```diff
@@ -242,6 +242,80 @@ validateCollateralEqBalance bal txcoll =
     SNothing -> pure ()
     SJust tc -> failureUnless (bal == tc) (UnequalCollateralReturn bal tc)
 
+-- > getValue txout ≥ inject ( ⌈ serSize txout ∗ coinsPerUTxOWord pp / 8 ⌉ )
+validateOutputTooSmallUTxO ::
+  ( Era era,
+    ToCBOR (Core.Value era),
+    HasField "_coinsPerUTxOWord" (Core.PParams era) Coin
+  ) =>
+  Core.PParams era ->
+  [Core.TxOut era] ->
+  Test (UtxoPredicateFailure era)
+validateOutputTooSmallUTxO pp outs = failureUnless (null outputsTooSmall) $ OutputTooSmallUTxO outputsTooSmall
+  where
+    Coin coinsPerUTxOWord = getField @"_coinsPerUTxOWord" pp
+    serSize = fromIntegral . BSL.length . serialize . getField @"value"
```

Comment:
let's add a comment here that we probably do not want to re-serialize the output, but rather use the length of the actual supplied bytes. this is a much trickier change, we should do it in a follow-up PR.